scanner/utils.go

```diff
@@ -13,30 +14,88 @@ import (
 	"github.com/bitrise-io/goinp/goinp"
 )
 
+func getDefaultValue(opt models.OptionNode) string {
+	if opt.Type == models.TypeOptionalSelector {
+		return ""
+	}
+
+	for key := range opt.ChildOptionMap {
+		return key
+	}
+	return ""
+}
+
+func getOptions(opts map[string]*models.OptionNode) (options []string) {
+	for key := range opts {
+		options = append(options, key)
+	}
+	return
+}
+
+func selectOption(options []string) (string, error) {
+	for i, option := range options {
+		fmt.Printf("[%d] : %s\n", i+1, option)
+	}
+	fmt.Printf("Type in the option's number, then hit Enter: ")
+
+	answer, err := goinp.AskForOptionalInput("", false)
```

Comment:
NOTE: pretty bed client-side interface: `AskForOptionalInput` optional = false.

Reply:
NOTE: this comment supposed to be in the package PR review